app/policies/document_policy.rb

```diff
@@ -8,6 +8,8 @@ def index?
   alias_method :edit?, :index?
   alias_method :update?, :index?
   alias_method :show?, :index?
+  #TODO fix this, attachments are using the wrong policy
```

Comment:
sadly yes. The attachments policy is completely unused by the entire application, at the moment. I'm not very familiar with how our permissions works, and didn't want to be digging into it too much